core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.spec.ts

```diff
@@ -64,138 +35,244 @@ describe('Translation Suggestion Review Modal Controller', function() {
   }));
 
   beforeEach(angular.mock.inject(function($injector, $controller) {
-    const $rootScope = $injector.get('$rootScope');
     contributionAndReviewService = $injector.get(
       'ContributionAndReviewService');
     SiteAnalyticsService = $injector.get('SiteAnalyticsService');
-
-    $uibModalInstance = jasmine.createSpyObj(
-      '$uibModalInstance', ['close', 'dismiss']);
-
     spyOn(
       SiteAnalyticsService,
       'registerContributorDashboardViewSuggestionForReview');
+  }));
+
+  describe('when reviewing suggestion', function() {
```

Comment:
I had to create two sections for this test one for the viewer and other for the reviewer. The reviewer section tests are the same as before.